newIDE/app/src/AssetStore/PrivateGameTemplates/PrivateGameTemplateInformationPage.js

```diff
@@ -0,0 +1,351 @@
+  grid: {
+    margin: '0 2px', // Remove the default margin of the grid but keep the horizontal padding for focus outline.
+  },
+};
+
+type Props = {|
+  privateGameTemplateListingData: PrivateGameTemplateListingData,
+  onOpenPurchaseDialog: () => void,
+  isPurchaseDialogOpen: boolean,
+  onGameTemplateOpen: PrivateGameTemplateListingData => void,
+|};
+
+const PrivateGameTemplateInformationPage = ({
+  privateGameTemplateListingData,
+  onOpenPurchaseDialog,
+  isPurchaseDialogOpen,
+  onGameTemplateOpen,
+}: Props) => {
+  const { id, name, sellerId } = privateGameTemplateListingData;
+  const { receivedGameTemplates } = React.useContext(AuthenticatedUserContext);
+  const [gameTemplate, setGameTemplate] = React.useState<?PrivateGameTemplate>(
+    null
+  );
+  const [isFetching, setIsFetching] = React.useState<boolean>(false);
+  const [
+    openSellerPublicProfileDialog,
+    setOpenSellerPublicProfileDialog,
+  ] = React.useState<boolean>(false);
+  const [
+    sellerPublicProfile,
+    setSellerPublicProfile,
+  ] = React.useState<?UserPublicProfile>(null);
+  const [
+    appStoreProductBeingBought,
+    setAppStoreProductBeingBought,
+  ] = React.useState(false);
+  const [errorText, setErrorText] = React.useState<?React.Node>(null);
+  const windowWidth = useResponsiveWindowWidth();
+  const isMobileScreen = windowWidth === 'small';
+
+  const isAlreadyReceived =
+    !!receivedGameTemplates &&
+    !!receivedGameTemplates.find(
+      gameTemplate => gameTemplate.id === privateGameTemplateListingData.id
+    );
+
+  React.useEffect(
+    () => {
+      (async () => {
+        setIsFetching(true);
+        try {
+          const [gameTemplate, profile] = await Promise.all([
+            getPrivateGameTemplate(id),
+            getUserPublicProfile(sellerId),
+          ]);
+
+          setGameTemplate(gameTemplate);
+          setSellerPublicProfile(profile);
+        } catch (error) {
+          if (error.response && error.response.status === 404) {
+            setErrorText(
+              <Trans>
+                Game template not found - An error occurred, please try again
+                later.
+              </Trans>
+            );
+          } else {
+            setErrorText(
+              <Trans>An error occurred, please try again later.</Trans>
+            );
+          }
+        } finally {
+          setIsFetching(false);
+        }
+      })();
+    },
+    [id, sellerId, privateGameTemplateListingData.appStoreProductId]
+  );
+
+  const onClickBuy = React.useCallback(
+    async () => {
+      if (!gameTemplate) return;
+      if (isAlreadyReceived) {
+        onGameTemplateOpen(privateGameTemplateListingData);
+        return;
+      }
+
+      try {
+        sendGameTemplateBuyClicked({
+          gameTemplateId: gameTemplate.id,
+          gameTemplateName: gameTemplate.name,
+          gameTemplateTag: gameTemplate.tag,
+        });
+
+        if (shouldUseAppStoreProduct()) {
```

Comment:
This check here is outdated, but it's my fault not to have backported a similar update in PrivateAssetPackInformationPage. I will do and let you know :)